### ISSUE TITLE
Autocomlete of functions eats closing bracket

### ISSUE THREAD

**Issue by echupriyanov** [joefitzgerald/go-plus#241]
Example:

```
fmt.Printf("Failed to lookup $HOST ip: %s\n", err.Er)
```

When I’m entering err.Err - I have correct suggestion Error(), but when I accept suggestion I’m getting following:

```
fmt.Printf("Failed to lookup $HOST ip: %s\n", err.Error()
```

i.e. I have err.Error() inserted, but my closing bracket is gone.

**Comment by caarlos0** [comment]
same here!

**Comment by joefitzgerald** [comment]
That sounds annoying! I'm sorry you're having this issue. I will look into it and try to reproduce.

**Comment by joefitzgerald** [comment]
Is this still an issue?

**Comment by joefitzgerald** [comment]
I think this is fixed in `v5.0.0` as a side effect of https://github.com/joefitzgerald/autocomplete-go/pull/50. Please let me know if you are still experiencing the issue and I will re-open.
